.github/workflows/tests-unit-php.yml

```diff
@@ -136,15 +136,15 @@ jobs:
         run: |
           if [[ $WP_VERSION == "5.7" && $PHP_VERSION == "7.2" ]]; then
             echo "Installing PHPUnit 7.5.x"
-            composer require --ignore-platform-reqs --no-interaction --no-scripts phpunit/phpunit:^7.5 --with-dependencies
+            composer require --no-interaction --no-scripts phpunit/phpunit:^7.5 --with-dependencies
           elif [[ $PHP_VERSION == "7.2" ]]; then
             echo "Installing PHPUnit 8.5.x"
-            composer require --ignore-platform-reqs --no-interaction --no-scripts phpunit/phpunit:^8.5 --with-dependencies
+            composer require --no-interaction --no-scripts phpunit/phpunit:^8.5 --with-dependencies
             echo "Downgrading phpunit/php-token-stream because the latest version requires PHP 7.3"
-            composer require --ignore-platform-reqs --no-interaction --no-scripts phpunit/php-token-stream:^3.1.3 --with-dependencies
```

Comment:
While we're at it, we can actually remove this part here because this condition is never met.